src/loaders/loader.h

```diff
@@ -30,7 +30,9 @@ int	libxmp_alloc_pattern		(struct xmp_module *, int);
 int	libxmp_alloc_track		(struct xmp_module *, int, int);
 int	libxmp_alloc_tracks_in_pattern	(struct xmp_module *, int);
 int	libxmp_alloc_pattern_tracks	(struct xmp_module *, int, int);
+#ifndef LIBXMP_CORE_PLAYER
 int	libxmp_alloc_pattern_tracks_long(struct xmp_module *, int, int);
+#endif
```

Comment:
IMO just let anything that tries to use these fail to link...

Reply:
It _will_ fail linkage in `lite` builds

Reply:
Oh, is this a Watcom thing? I haven't seen GCC fail to link due to undefined function declarations that aren't used.

Reply:
I actually meant any internal use of the procedure will fail linkage.
Otherwise, this (and the other non-`LIBXMP_CORE_PLAYER` procs)
are not public procs: they won't be used by any clients either way.
(Am I reading you correctly?)

Reply:
Oh, I see. What I meant is that leaving the declarations as-is and letting linking fail if anyone actually tries to use them (which *shouldn't* currently be happening in `lite` builds) would probably be okay. It doesn't really matter that much though. Let's just leave the #ifdefs in the headers.

Reply:
OK